### PR TITLE
FEI-4966.1: Remove file extensions from all imports/requires

### PR DESCRIPTION
**Pull request by kevinbarabash**
## Summary:
Before migrating wonder-blocks files to TypeScript, we need to remove the file extensions from all of the imports since TypeScript doesn't work with them.  This PR also updates the eslint config to disallow file extensions on imports moving forward.

Issue: FEI-4966

## Test plan:
- let CI run lint

### CODE REVIEW

**Review by jeresig — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "flow": "flow",
     "jest": "jest --config config/jest/test.config.js",
     "lint:watch": "esw --watch --config ./eslint/eslintrc packages",
-    "lint": "eslint --config .eslintrc.js packages",
+    "lint": "eslint --config .eslintrc.js .",
```
Comment (kevinbarabash) [inline package.json:22]: I'm going to revert this change.
